src/openapi.json

```diff
@@ -2621,6 +2659,61 @@
 					}
 				}
 			}
+		},
+		"/users/{userKeycloakUserId}/changemakers/{changemakerId}/permissions/{permission}": {
```

Comment:
This is currently added in the first commit (correct user type); does it belong in the second (add put endpoint)?

Reply:
Good catch thanks